l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -278,12 +278,19 @@ def _prepare_generic_line_data(self, line):
                     ('type_tax_use', '=', 'purchase'),
                     ('kind_id.code', '=', line.Natura),
                     ('amount', '=', 0.0),
-                ], order='sequence', limit=1)
+                ], order='sequence')
             if not account_taxes:
                 self.log_inconsistency(
                     _('No tax with percentage '
                       '%s and nature %s found. Please configure this tax.')
                     % (line.AliquotaIVA, line.Natura))
+            if len(account_taxes) > 1:
+                self.log_inconsistency(
+                    _('Too many taxes with percentage '
+                      '%s and nature %s found. Tax %s with lower priority has '
+                      'been set on invoice lines.')
+                    % (line.AliquotaIVA, line.Natura,
+                       account_taxes[0].description))
         else:
```

Comment:
qui manca https://github.com/OCA/l10n-italy/pull/1096/files#diff-a1b600a217458c87c2b54a559199629aR300 , l'hai tolta per un motivo specifico?

Reply:
Non mi pare necessario, in quanto alla fine viene comunque usato `account_taxes[0]`:
https://github.com/OCA/l10n-italy/pull/1148/files#diff-a1b600a217458c87c2b54a559199629aR327

Reply:
vero